buildSrc/settings.gradle.kts

```diff
@@ -1,3 +1,8 @@
+
 dependencyResolutionManagement {
-  versionCatalogs { create("libs") { from(files("../gradle/libs.versions.toml")) } }
+  versionCatalogs {
+    create("libs") {
```

Comment:
so we were not using it b4 although the catalog was present, right?

Reply:
Hmm there was no `buildSrc` until recently!

Reply:
Yes, exactly, there were only hardcoded references to libs in buildSrc kotlin code